controllers/grid/submissions/ExportPublishedSubmissionsListGridCellProvider.inc.php

```diff
@@ -51,12 +51,12 @@ function getCellActions($request, $row, $column, $position = GRID_ACTION_POSITIO
 				if (empty($title)) $title = __('common.untitled');
 				$authorsInTitle = $publishedSubmission->getShortAuthorString();
 				$title = $authorsInTitle . '; ' . $title;
-				import('lib.pkp.controllers.grid.submissions.SubmissionsListGridCellProvider');
+				import('lib.pkp.classes.core.ServicesContainer');
 				return array(
 					new LinkAction(
 						'itemWorkflow',
 						new RedirectAction(
-							SubmissionsListGridCellProvider::getUrlByUserRoles($request, $publishedSubmission)
+							ServicesContainer::instance()->getWorklowUrlByUserRoles($publishedSubmission)
```

Comment:
This was attached to the old submissions grids. I moved it over to the SubmissionService when I got rid of the old grids:

https://github.com/NateWr/pkp-lib/blob/i2163_js_with_api/classes/services/SubmissionService.php#L93